addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3635,7 +3634,19 @@ msgctxt "#844"
 msgid "Deactivate"
 msgstr ""
 
-#empty strings from id 845 to 849
+#. Message in a dialog when a user wants to delete a repeating timer
+#: xbmc/pvr/windows/GUIWindowPVRBase.cpp
+msgctxt "#845"
+msgid "Are you sure you want to delete this repeating timer, including all timers it has scheduled?"
```

Comment:
> I was never a big fan of "repeating timer"

@da-anda I don't really care and will accept whatever is most appropriate in the eyes of native English speakers. But currently its called repeating timer everywhere and introducing new terminology in only one string leads to inconsistency. If we find agreement on another wording (timer schedule, recording rule, ... whatever) we should adjust all occurrences at once to stay consistent - in another PR.